### ISSUE TITLE
TypeError: Class constructor GraphQLList cannot be invoked without 'new'

### ISSUE THREAD

**Issue by justinwilaby** [mswjs/data#258]
When targeting ES6, the following error is output to the console:

```
TypeError: Class constructor GraphQLList cannot be invoked without 'new'
    at generateGraphQLHandlers.js:100:1
    at Array.reduce (<anonymous>)
    at createComparatorGraphQLInputType (generateGraphQLHandlers.js:99:1)
    at ./node_modules/@mswjs/data/lib/model/generateGraphQLHandlers.js (generateGraphQLHandlers.js:107:1)
    at __webpack_require__ (bootstrap:19:1)
    at ./node_modules/@mswjs/data/lib/factory.js (factory.js:41:33)
    at __webpack_require__ (bootstrap:19:1)
    at ./node_modules/@mswjs/data/lib/index.js (index.js:11:17)
    at __webpack_require__ (bootstrap:19:1)
    at ./src/mocks/handlers/sxxxxr.db.ts (xxxxxxx.ts:122:3)
 ```

Adding `new` at the line specified fixes the issue.

**Comment by kettanaito** [comment]
Hey, @justinwilaby. Thanks for reporting this.

I confirm this is a bug on this line:

https://github.com/mswjs/data/blob/efb17476bc8fedb759b4f0977bc2b96df6ebc0ff/src/model/generateGraphQLHandlers.ts#L60

Will release the fix later this week.

**Comment by kettanaito** [comment]
## Released: v0.11.1 🎉

This has been released in v0.11.1!

- 📄 [**Release notes**](https://github.com/mswjs/data/releases/tag/v0.11.1)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/data/v/0.11.1)

Make sure to always update to the latest version (`npm i @mswjs/data@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Comment by justinwilaby** [comment]
Thank you for the fast response!
